OpenRA.Mods.Common/Lint/CheckDefaultVisibility.cs

```diff
@@ -36,6 +36,9 @@ static void Run(Action<string> emitError, Ruleset rules)
 				// Catch TypeDictionary errors.
 				try
 				{
+					if (LintExts.ActorIsPlayer(actorInfo.Key) || LintExts.ActorIsWorld(actorInfo.Key))
```

Comment:
```
if (Enum.TryParse<SystemActors>(actorInfo.Key, true, out _))
	continue;
```
would IMO be cleaner, and matches the behaviour of the `CheckTraitLocation` check.

Reply:
makes sense